common/errors.go

```diff
@@ -18,7 +18,7 @@
 // OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
 // THE SOFTWARE.
 
-package repository
+package common
```

Comment:
Consider using `codegen/errors/errors.go` instead of `common/errors.go` to reduce number of top level folders in the git repo.